docs/fireblocks.md

```diff
@@ -0,0 +1,92 @@
+# Fireblocks integration
+
+## Description
+
+The Fireblocks integration allows you to connect your wallet with Fireblocks using the RAW transaction signing method.
+This means that the transaction is managed by the headless wallet and not by the Fireblocks API or dashboard.
+
+## Configuration
+
+The configuration requires an api key and secret which can be created on the Fireblocks dashboard.
+Under `Settings > Users` you can create a new API user with the `Editor` role and copy the API key and secret.
```

Comment:
suggestion: I'd recommend adding a link to one of their own tutorials about the Fireblocks service. Those navigation instructions can become obsolete and it can take a while for us to notice and to update this document.

Reply:
Yes, but they don't have one for this.
RAW signing is a way to externally manage the wallet (which is what we are doing) so it depends on how the integration will be done, this is why I think they didn't document this.